src/Input/Input.js

```diff
@@ -6,7 +6,7 @@ import classNames from 'classnames';
 import { createStyleSheet } from 'jss-theme-reactor';
 import customPropTypes from '../utils/customPropTypes';
 
-function isDirty(obj) {
+export function isDirty(obj) {
```

Comment:
Perhaps this could live here: https://github.com/callemall/material-ui/blob/next/src/utils/helpers.js ?